### PR TITLE
fixes for new MTK

### PR DESCRIPTION
**Pull request by shashi**

### CODE REVIEW

**Comment by shashi** [comment]
latex test fails but it should be fixed in MTK @YingboMa watch for this fail... It's basically trying to write `x(t) * 1` as `*(x(t), 1)`. I think we should fix it in MTK using `Base.isoperator`.

**Review by YingboMa — APPROVED** [review]
LGTM

File: src/utils.jl [hunk 1]
```diff
@@ -18,24 +18,24 @@ function build_component_funcs(symex)
...
+function modelingtoolkitize_expr(ex::Sym,vars,curmod)
+    ex
 end
 
 function modelingtoolkitize_expr(ex::Symbol,vars,curmod)
-    names = [x.op.name for x in vars]
-    op = ex ∈ names ? vars[findfirst(x->ex == x.op.name,vars)] : getproperty(curmod,ex) # HACK
-    convert(Expression,op)
+    names = tosymbol.(vars)
+    op = ex ∈ names ? vars[findfirst(x->(ex == tosymbol(x)),vars)] : getproperty(curmod,ex) # HACK
```
Comment (YingboMa) [inline src/utils.jl:35]:
```suggestion
    op = ex ∈ names ? vars[findfirst(x->ex == x, names)] : getproperty(curmod,ex)
```
I think this also works?

**Review by ChrisRackauckas — APPROVED** [review]

**Comment by ChrisRackauckas** [comment]
We're going to need to follow up and fix the Latexify tests @korsbo

**Pull request merged**